test-configs.yaml

```diff
@@ -461,6 +461,24 @@ device_types:
     filters:
       - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
 
+  meson_g12a_sei510:
+    name: 'meson-g12a-sei510'
+    mach: amlogic
+    class: arm64-dtb
+    boot_method: uboot
+    flags: ['little_endian']
```

Comment:
Like I wrote before, we shouldn't be introducing a `little_endian` flag now.  Little-endian is the default unless big-endian is specified.